src/RadialGauge/RadialGaugeSeries/StackedRadialGaugeSeries.tsx

```diff
@@ -63,6 +67,14 @@ export interface StackedRadialGaugeSeriesProps {
     typeof StackedRadialGaugeValueLabel
   > | null;
 
+  /**
+   * Supporting label component.
+   */
+    supportingLabel: ReactElement<
```

Comment:
I'm not in love with this name but i don't have a better suggestion right now. Let me think on it.

Reply:
How about `descriptionLabel` instead?

Reply:
Updated.